app/assets/stylesheets/screen.scss

```diff
@@ -706,3 +695,33 @@ div#keycut-help {
     padding-left: 45px;
   }
 }
+
+.fa-star {
+  color: #FFE000;
+}
+.fa-star-o {
+  color: #726904;
+}
+.fa-icon-overlay {
+  position: relative;
+}
+.fa-icon-overlay .fa {
+  position: absolute;
+  top: 0;
+  left: 0;
+}
+.fa-icon-overlay .fa:first-child {
+  position: initial;
+}
+.fa-comment {
+  color: white;
+}
+.fa-comment-o {
+  color: gray;
+}
+.fa-bug {
+  color: #C20B0B;
+}
+.fa-cog {
+  color: #5E5C5C;
+}
```

Comment:
other thing, maybe should be nice put this in a separated file, and use as start to organize the stylesheets :)